application/models/Traits/PermissionTrait.php

```diff
@@ -31,6 +31,41 @@ public static function getMinimalPermissionRead()
         return null;
     }
 
+    /**
+     * Get minimal access criteria : user can access to object
```

Comment:
Can we duplicate this comment and add it in the interface?
Or, as to no duplicate, move it and add a reference here saying "more info can be found on the interface"?

Reply:
.** @inheritdoc */ in Trait ?

Reply:
Not sure... :) Not sure how to handle this situation